code/game/machinery/computer/crew.dm

```diff
@@ -137,9 +137,13 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 
 	var/list/results = list()
 
-	for(var/mob/living/carbon/human/H in GLOB.carbon_list)
+	for(var/mob/living/carbon/human/H as() in GLOB.suit_sensors_list)
+		if(!H)
+			stack_trace("Null reference in suit sensors list")
 
```

Comment:
This should remove H from the suit sensors list so it doesn't cause a stack trace every single time